### ISSUE TITLE
[Critical] Function cannot be performed by keyboard alone

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#580]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87724 |
| **Checkpoint** | 2.1 Keyboard Accessible › Keyboard Navigation (2.1.1.a) |
| **Impact** | Critical |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 01 Settings Workflow |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/11/26 at 12:20 pm |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

There is no way to perform the function using only the keyboard on the same screen or on a qualifying conforming alternate version.
In Settings menu, select BagIt Profiles, then New, then accept the default of AP Trust, then select Create. Expand the Tag Files, the Add New Tag File cannot be selected using keyboard alone. See Settings User Experience video at 11:40.

## Recommendation

Rule : 
Functionality MUST be available using the keyboard, unless the functionality cannot be accomplished in any known way using a keyboard.

How To Fix: 
Fix this issue by ensuring the component can be used by the keyboard. This includes ensuring controls are:
1. Focusable with a keyboard (i.e. can be reached using the Tab key or Arrow keys on a keyboard), and
2. Operable with a keyboard (e.g. links are activated with the Enter key, buttons are activated with Enter and Spacebar, checkboxes are activated with Spacebar, dragging and swiping actions have a keyboard alternative, etc.)

The best way to do this is to use native HTML elements when HTML equivalents exist (e.g. anchor, button, select, text area, and input elements). When custom (non native HTML) controls have to be used, you must ensure they can be focused with a keyboard (often using tabindex="0") and operated with a keyboard (using JavaScript event handlers). The ARIA Authoring Practices Guide provides information on expected keyboard interaction patterns for custom controls and widgets and working examples (see References below).

Reference: 
ARIA Authoring Practices Guide Patterns: https://www.w3.org/WAI/ARIA/apg/patterns/

Background: 
Some people cannot use a mouse due to vision or motor disabilities. Content that can be operated with a mouse must also be made operable with a keyboard. When content is operable through a keyboard, it becomes operable by a variety of assistive technologies such as speech input software, sip-and-puff software, on-screen keyboards, scanning software, and alternate keyboards.

## Screenshots

![01_keyboard3](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87724/01_keyboard3.png)
